pilot/pkg/bootstrap/options.go

```diff
@@ -69,6 +69,7 @@ type PilotArgs struct {
 	DiscoveryOptions         DiscoveryServiceOptions
 	InjectionOptions         InjectionOptions
 	ValidationOptions        ValidationOptions
+	PodName                  string
```

Comment:
by default hostname equals podname.

Reply:
@howardjohn  looks like you're just using pod name for `Identity` in the election config? What are the requirements for this identity? Does it have to be a pod name?